lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -25,10 +25,24 @@ const axe = fs.readFileSync(
 );
 
 // This is run in the page, not Lighthouse itself.
+// axe.run returns a promise which fulfills with a results object
+// containing any violations.
 /* istanbul ignore next */
 function runA11yChecks() {
-  return new Promise((resolve, reject) => {
-    axe.a11yCheck(document, resolve);
+  return axe.run(document, {
+    runOnly: {
+      type: 'rule',
+      values: [
+        'aria-allowed-attr',
```

Comment:
yeah there are actually predefined rulesets we could use, for instance, run all the rules required for WCAG A:
```
{
  runOnly: {
      type: "tag",
      values: ["wcag2a"]
    }
}
```

I would like to actually move over to doing that as the next step. But I think that will also require reworking the audits and tests a bit. So before doing that I thought it might be nice to just make lighthouse do less work in the meantime.